Cohort2/team4/server/src/routers/event.ts

```diff
@@ -0,0 +1,80 @@
+import { router, publicProcedure } from '../trpc';
+import { Sequelize } from 'sequelize-typescript';
+import Events from '../models/Event';
+import { z } from 'zod';
+import { randomUUID } from 'crypto';
+
+
+type Event = {
+  name: String,
+  short_description: String,
+  long_description: String,
+  city: String,
+  state: String,
+  country: String,
+  address: String,
+  start_time: Date,
+  end_time: Date,
+  host: number,
+};
+
+const EVENTS: Event[] = [
```

Comment:
Why is this not hooked into the DB? I thought sequelize was setup?

Reply:
I see that you followed Michelle's example. Ok that works. We can do the DB next.